TWTValidation/JSON Validator/Schema Validators/TWTJSONSchemaArrayValidator.m

```diff
@@ -152,12 +156,16 @@ - (BOOL)validateValue:(id)value error:(out NSError *__autoreleasing *)outError
             if (index < self.indexedItemValidators.count) {
                 if (![self.indexedItemValidators[index] validateValue:item error:outError ? &error : NULL]) {
                     itemsValidated = NO;
-                    //                        add error
+                    if (outError) {
```

Comment:
yep... I'd assumed that if `outError != nil` and so we passed in `&error` to the validator, we'd get an error back. But it should be `if (error)` since the real purpose is to prevent passing in nil to `addObject:`.